ci/test_integrate_easy.py

```diff
@@ -0,0 +1,177 @@
+    parser.add_argument(
+        "--run_collect_stats",
+        action="store_true",
+        help="path to train sentencepiece model",
+    )
+    parser.add_argument(
+        "--run_train",
+        action="store_true",
+        help="path to train sentencepiece model",
+    )
+    args = parser.parse_args()
+
+    # In this test we assume that we use mini_an4 dataset.
+    data_info = {
+        "speech": ["wav.scp", "sound"],
+        "text": ["text", "text"],
+    }
+
+    # Train sentencepiece model
+    if args.task == "lm":
+        user_defined_symbols = ["<generatetext>"]
+        data_info.pop("speech")
+    else:
+        user_defined_symbols = []
+
+    # Tokenize if tts
+    if args.task == "tts" or args.task == "gan_tts":
+        ez.preprocess.prepare_sentences(
+            [args.train_dump_path / "text"], args.data_path / "tokenize"
+        )
+        ez.preprocess.tokenize(
+            input=str(args.data_path / "tokenize" / "train.txt"),
+            output=str(args.train_dump_path / "tokenize" / "tokenized.txt"),
+            token_type="phn",
+            cleaner="tacotron",
+            g2p="g2p_en",
+        )
+        if args.train_sentencepiece_model:
+            ez.preprocess.train_sentencepiece(
+                args.train_dump_path / "tokenize" / "tokenized.txt",
+                args.data_path / "spm/bpemodel",
+                vocab_size=50,
+                character_coverage=0.9995,
+                user_defined_symbols=user_defined_symbols,
+            )
+
+    elif args.train_sentencepiece_model:
+        ez.preprocess.prepare_sentences(
+            [args.train_dump_path / "text"], args.data_path / "spm"
+        )
+        ez.preprocess.train_sentencepiece(
+            args.data_path / "spm/train.txt",
+            args.data_path / "spm/bpemodel",
+            vocab_size=50,
+            character_coverage=0.9995,
+            user_defined_symbols=user_defined_symbols,
+        )
+
+    # Prepare configurations
+    exp_dir = str(args.exp_path / args.task)
+    stats_dir = str(args.exp_path / "stats")
+
+    training_config = ez.config.from_yaml(args.task, args.config_path)
+    training_config["max_epoch"] = 1
+    training_config["ngpu"] = 0
+    training_config["bpemodel"] = str(args.data_path / "spm/bpemodel/bpe.model")
+    with open(args.data_path / "spm/bpemodel/tokens.txt", "r") as f:
+        tokens = [t.replace("\n", "") for t in f.readlines()]
+        training_config["token_list"] = tokens
+
+    if args.task == "tts" or args.task == "gan_tts":
+        normalize = training_config["normalize"]
+        training_config["normalize"] = None
+        pitch_normalize = training_config["pitch_normalize"]
+        training_config["pitch_normalize"] = None
+        energy_normalize = training_config["energy_normalize"]
+        training_config["energy_normalize"] = None
+
+    # if args.run_config is not None:
+    #     training_config.update(parser.parse_args(shlex.split(args.run_config)))
```

Comment:
Do you want to keep these lines?
If so, please add some comments.

Reply:
These lines are not required; I will remove them.